rakefile.netstd.rb

```diff
@@ -99,7 +100,7 @@
 end

```

Comment:
I don't know how to make a comment on an unchanged line. Look at
* line 52: "teamcity_server_url"
* line 59: "if necessary, change `VERSION_SUFFIX` on the TeamCity"

And in rakefile.rb, we don't have the same changes, but that's because our instructions differ.
TeamCity still appears, though.